readability improvements (#1278)
openhab/openhab-docs: introduction.md

If you have a strong preference towards a particular platform, then that platform is probably your best choice.

You can install openHAB on your desktop computer for evaluation purposes if you already have any of these systems available for use, but we recommend using a dedicated system in the long run. 
If you feel serious about home automation it is better to start with a dedicated system right away.

If you have no strong preference, get a [Raspberry Pi 4](https://www.raspberrypi.org/products/raspberry-pi-4-model-b/) with 2 GB and a 16 GB SD card and install [openHABian]({{base}}/installation/openhabian.html) for the best experience.

> A RPi 2 or 3 is fine, too, but a RPi 4 with 8GB is overkill, as are larger SD cards.

Reviewer: ```suggestion
A RPi 2 or 3 is fine, too, but a RPi 4 with 8GB of RAM is overkill, as are larger SD cards.
```

> While openHABian offers a streamlined and simplified way to get up and running quickly, it is a complete openHAB eco system in a box and easily capable of automating your entire home.

Reviewer: ```suggestion
While openHABian offers a streamlined and simplified way to get up and running quickly, it is a complete openHAB ecosystem in a box and easily capable of automating your entire home.
```

> Mind two potential limitations of Raspberry Pis:

Reviewer: ```suggestion
Please be aware of two potential limitations of Raspberry Pis:
```

RPi 3 and older are limited in RAM (1 GB of memory or less) and will probably not perform well when additional memory hungry applications such as databases and data visualization programs are being used.
You will want to have two GB for that to work smoothly.
Running Raspberries off the internal SD card only may result in system instabilities as these memory cards can degrade quickly under openHAB's use conditions (infamous 'wearout').
When you choose to deploy openHABian, it'll use the ZRAM feature to mitigate.

Once you have openHAB up and running, the [Configuration]({{base}}/configuration/index.html) article contains everything you need to know to get your openHAB installation talking to different devices around your home. 

You will quickly discover that you may want to learn more about Things, Channels, Items, and more. 
To do so, we highly recommend that you read the next chapter titled [Concepts]({{base}}/concepts/index.html). 
It provides more in-depth descriptions of Things, Items, Bindings, etc. that will help you as you dive deeper into openHAB.

The amount of information provided can be overwhelming but don't hesitate to return here to lookup infos.

> When you encounter questions along the way that the docs don't answer for you, the openHAB community is there to help.

Reviewer: ```suggestion
When you encounter questions along the way that the docs don't answer for you, the openHAB community is here to help.
```
